src/js/otus/sagas.js

```diff
@@ -109,22 +111,18 @@ export function* removeSequence(action) {
 
 export function* revert(action) {
     try {
-        yield otusAPI.revert(action);
+        yield otusAPI.revert(action.payload);
 
-        if (action.otuVersion === 0) {
+        if (action.payload.otuVersion === 0) {
             const path = yield select(getCurrentOTUsPath);
             yield put(push(path));
         } else {
-            const otuResponse = yield otusAPI.get(action);
-            const historyResponse = yield otusAPI.getHistory(action);
-            yield put({
-                type: REVERT.SUCCEEDED,
-                data: otuResponse.body,
-                history: historyResponse.body
-            });
+            const otuResponse = yield otusAPI.get(action.payload);
+            const historyResponse = yield otusAPI.getHistory(action.payload);
```

Comment:
Could we use [`all`](https://redux-saga.js.org/docs/api#alleffects---parallel-effects) here?

Reply:
Neat, looks like yes. Should be a decent time save